packages/scoutgame/src/projects/updateScoutProject.ts

```diff
@@ -94,19 +114,31 @@ export async function updateScoutProject(payload: UpdateScoutProjectFormValues,
     payloadProjectContractAddresses.includes(address)
   );
 
+  const deletedWalletAddressesToRestore = deletedWalletAddresses.filter((address) =>
+    payloadProjectWalletAddresses.includes(address)
+  );
+
   const retainedProjectMemberIds = projectMemberIds.filter((scoutId) => payloadProjectMemberIds.includes(scoutId));
 
   const builderMemberIds = [...retainedProjectMemberIds, ...projectMemberIdsToRestore, ...projectMemberIdsToCreate];
 
-  const builderMembersCount = await prisma.scout.count({
+  const builderMembers = await prisma.scout.findMany({
     where: {
-      builderStatus: 'approved',
       id: {
         in: builderMemberIds
       }
+    },
+    select: {
+      id: true,
+      utmCampaign: true,
+      builderStatus: true
     }
   });
 
+  const builderMembersCount = builderMembers.filter(
```

Comment:
it'd be cleaner i think if u just updated the query to use an OR condition:

where: {
  OR: [{ builderStatus: 'approved' }, { utmCampaign: 'taiko' }]
  ....
  }